lib/models/widget_model.dart

```diff
@@ -16,26 +17,117 @@ enum WidgetStatus {
   NONE
 }
```

Comment:
**[IDEA]** maybe this can be part of `/config.json`

Reply:
I think current solution is pretty good